yt_dlp/extractor/nebula.py

```diff
@@ -478,3 +481,48 @@ def _real_extract(self, url):
             playlist_id=collection_slug,
             playlist_title=channel.get('title'),
             playlist_description=channel.get('description'))
+
+
+class NebulaSeasonIE(NebulaBaseIE):
+    IE_NAME = 'nebula:season'
+    _VALID_URL = rf'{_BASE_URL_RE}/(?P<series>[\w-]+)/?season/(?P<season_number>\d+)'
+    _TESTS = [{
+        'url': 'https://nebula.tv/jetlag/season/15',
+        'info_dict': {
+            'id': 'jetlag_15',
+        },
+        'playlist_count': 8,
+    }, {
+        'url': 'https://nebula.tv/jetlag/season/14',
+        'info_dict': {
+            'id': 'jetlag_14',
+        },
+        'playlist_mincount': 8,
+    }]
+
+    def _entries(self, data):
+        base_url = 'https://nebula.tv/videos'
+        for episode in traverse_obj(data, ('episodes')):
+            episode_id = traverse_obj(episode, ('video', 'id'))
+            metadata = self._extract_video_metadata(episode)
+            yield self.url_result(smuggle_url(f'{base_url}/{episode_id}', {'id': episode_id}),
+                                  NebulaIE, episode_id, url_transparent=True, **metadata)
```

Comment:
Now we can use the `_build_url_result` method I suggested above:
```suggestion
            yield self._build_url_result(episode)
```